packages/utils/upgrade/src/modules/file-scanner/scanner.ts

```diff
@@ -11,7 +11,10 @@ export class FileScanner implements FileScannerInterface {
   }
 
   scan(patterns: string[]) {
-    const filenames = glob.sync(patterns, { cwd: this.cwd });
+    // we use fastglob instead of glob because it supports negation patterns
```

Comment:
did you know you can pass an ignore param to glob directly ?

Reply:
Yeah, I wanted to keep it all in one place in `paths` so that we know what to expect